modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncTask.java

```diff
@@ -41,7 +41,7 @@ public void run() {
         }
     }
 
-    protected abstract void execute();
+    protected abstract void execute() throws Exception;
```

Comment:
Don't do this

Reply:
Unfortunately it is necessary because the implementation in the TranslationTask throws a InterruptedException, I didn't think having a try catch in the TranslationTask method would be good because the error will likely never be thrown and AsyncTask already has a try catch for exceptions, I could change it to throws InterruptedException but that would be the best fix.

Reply:
Ya you should be as explicit as possible with exceptions